webpack.config.js

```diff
@@ -1,37 +1,37 @@
 /* eslint-disable @typescript-eslint/explicit-function-return-type */
 /* eslint-disable @typescript-eslint/camelcase */
 /* eslint-disable @typescript-eslint/no-var-requires */
-const path = require('path');
 const TerserPlugin = require('terser-webpack-plugin');
-const config = require('./packman.config.json');
-
-const entries = {};
-config.function.forEach(f => {
-    entries[f.name] = path.resolve(__dirname, f.source.location);
-});
+const path = require('path');
 
 module.exports = {
-    entry: entries,
+    entry: {
```

Comment:
I got this working with multiple entries... need to add entries and use `splitChunks` in the optimization section for webpack.